Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                var firstName = InputName("First name: ");
+
+                var surName = InputName("Surname: ");
+
+                var age = InputInteger("Age: ");
+
+                var weightInKilograms = InputFloat("Weight in kilograms: ");
+
+                var heightInCentimeters = InputFloat("Height in centimeters: ");
+
+                Console.WriteLine($"{firstName} {surName} is {age} years old, his weight is {weightInKilograms} kg and his height is {heightInCentimeters} cm.");
+
+                var bodyMassIndex = CalculateBodyMassIndex((float)weightInKilograms, (float)heightInCentimeters);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary cast to 'float'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845598)